internal/dag/builder_test.go

```diff
@@ -1160,6 +1160,39 @@ func TestDAGInsertGatewayAPI(t *testing.T) {
 			),
 		},
 		"insert basic single route, single hostname, gateway with TLS": {
+			gateway: &gatewayapi_v1alpha1.Gateway{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "contour",
+					Namespace: "projectcontour",
+				},
+				Spec: gatewayapi_v1alpha1.GatewaySpec{
+					Listeners: []gatewayapi_v1alpha1.Listener{{
+						Port:     443,
+						Protocol: gatewayapi_v1alpha1.HTTPProtocolType, // <--- invalid protocol, must be "HTTPS"
```

Comment:
is this comment removable/movable to the below test case?

Reply:
or maybe the test case names got mixed up? looking at the full file it looks like this test case is actually `"insert basic single route, single hostname, gateway with TLS, invalid listener protocol"`

Reply:
I'll remove the comment since now the protocol is ignored in relation to if tls is defined or not.